include/zephyr/net/tftp.h

```diff
@@ -132,36 +134,38 @@ struct tftpc {
 	uint8_t tftp_buf[TFTPC_MAX_BUF_SIZE];
 };
 
-/* @brief This function gets data from a "file" on the remote server.
+/**
+ * @brief This function gets data from a "file" on the remote server.
  *
  * @param client      Client information of type @ref tftpc.
  * @param remote_file Name of the remote file to get.
  * @param mode        TFTP Client "mode" setting.
  *
- * @return The size of data being received if the operation completed successfully.
- *         TFTPC_BUFFER_OVERFLOW if the file is larger than the user buffer.
- *         TFTPC_REMOTE_ERROR if the server failed to process our request.
- *         TFTPC_RETRIES_EXHAUSTED if the client timed out waiting for server.
- *         -EINVAL if `client` is NULL.
+ * @retval The size of data being received if the operation completed successfully.
+ * @retval TFTPC_BUFFER_OVERFLOW if the file is larger than the user buffer.
+ * @retval TFTPC_REMOTE_ERROR if the server failed to process our request.
+ * @retval TFTPC_RETRIES_EXHAUSTED if the client timed out waiting for server.
+ * @retval -EINVAL if `client` is NULL.
```

Comment:
This likely needs a closer inspection, it does not seem ok that an API mixes error codes from two namespaces. Not something to be fixed in this PR of course.